contracts/bonding/BondingManager.sol

```diff
@@ -501,7 +501,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
             address electedTranscoder = availableTranscoders[numAvailableTranscoders - 1];
 
             // Pseudorandomly pick an available transcoder weighted by its stake relative to the total stake of all available transcoders
-            uint256 r = uint256(block.blockhash(block.number - 1)) % totalAvailableTranscoderStake;
+            uint256 r = uint256(block.blockhash(_block)) % totalAvailableTranscoderStake;
```

Comment:
This means we have to set the round time to be greater than 256 blocks right?  Otherwise would `totalAvailableTranscoderStake` change between job creation and claim?